bin/alluxio

```diff
@@ -237,7 +237,7 @@ function main {
           # if ufs is local filesystem and already exists
           exit 0
         else
-          declare -a schemes=(hdfs s3 s3n glusterfs swift oss gcs)
+          declare -a schemes=(hdfs s3 s3a s3n glusterfs swift oss gcs)
```

Comment:
For s3 I've changed it to s3a by default, and added s3n and s3a explicit options to bootstrap conf.